util/kustomize/kustomize.go

```diff
@@ -271,23 +281,24 @@ func (k *kustomize) Build(opts *v1alpha1.ApplicationSourceKustomize, kustomizeOp
 			}
 			updatedKustomization, err := yaml.Marshal(kMap)
 			if err != nil {
-				return nil, nil, fmt.Errorf("failed to marshal kustomization.yaml after adding patches: %w", err)
+				return nil, nil, nil, fmt.Errorf("failed to marshal kustomization.yaml after adding patches: %w", err)
 			}
 			kustomizationFileInfo, err := os.Stat(kustomizationPath)
 			if err != nil {
-				return nil, nil, fmt.Errorf("failed to stat kustomization.yaml: %w", err)
+				return nil, nil, nil, fmt.Errorf("failed to stat kustomization.yaml: %w", err)
 			}
 			err = os.WriteFile(kustomizationPath, updatedKustomization, kustomizationFileInfo.Mode())
 			if err != nil {
-				return nil, nil, fmt.Errorf("failed to write kustomization.yaml with updated 'patches' field: %w", err)
+				return nil, nil, nil, fmt.Errorf("failed to write kustomization.yaml with updated 'patches' field: %w", err)
 			}
+			commands = append(commands, "# kustomization.yaml updated with patches. There is no `kustomize edit` command for this, so you will need to copy the patches into kustomization.yaml manually.")
```

Comment:
> "There is no `kustomize edit` command for this"

Maybe rephrase this piece? Not sure what "this" is referring to. 

> "so you will need to copy the patches into kustomization.yaml"

I am not having the big picture about how the list of commands will be displayed but I am missing the contextualization about why this message is required. Maybe start with something like: "In order to generate the manifests in your development environment, it is necessary to copy the patches..."?

Reply:
Updated the message to be a bit more clear.